doc/languages-frameworks/go.section.md

```diff
@@ -176,13 +176,13 @@ Following example could be used to only build the example-cli and example-server
 
 Specified as a string or list of strings. Causes the builder to skip building child packages that match any of the provided values.
 
-### `CGO_ENABLED` {#var-go-CGO_ENABLED}
+### `env.CGO_ENABLED` {#var-go-CGO_ENABLED}
```

Comment:
There is a section about "Controlling the Go environment" (L248). Which is hinting at GOFLAGS and the other things you'd like to see in `env.`. Maybe we can somehow join these? Preferably move the that sec directly under the var section and env.CGO_ENABLED into it.

Reply:
Cool!

Reply:
I moved the paragraphs under `var-go-CGO_ENABLED` under the environment variables subsection and mentioned using the `env` attribute.